lib/util/git/index.ts

```diff
@@ -249,9 +249,9 @@ export async function initRepo(args: StorageConfig): Promise<void> {
 
 async function resetToBranch(branchName: string): Promise<void> {
   logger.debug(`resetToBranch(${branchName})`);
-  await git.raw(['reset', '--hard']);
+  await git.raw(['reset', '--hard', 'HEAD']);
```

Comment:
I believe this change is functionally identical to the original. I'm ok with changing it, but have you identified any reason for this change?

Reply:
No, that's correct. It's identical. The reason why I added `HEAD` is to make it explicit and the purpose clearer.

I can revert if you want.

Reply:
I think let's revert if we think it's working as is

Reply:
Done.